--- slang-2.2.4/configure.norpath~	2011-04-11 04:37:07.000000000 +0200
+++ slang-2.2.4/configure	2012-03-07 14:46:14.328943741 +0100
@@ -6082,7 +6082,7 @@ ELF_CFLAGS="$ELF_CFLAGS $IEEE_CFLAGS"
 CFLAGS="$CFLAGS $IEEE_CFLAGS"
 
 
-if test "X$libdir" != "X"
+if test "X$libdir" != "X/usr/lib" -a "X$libdir" != "X/usr/lib64" -a "X$libdir" != "X"
 then
   if test "X$RPATH" = "X"
   then
@@ -7050,7 +7050,7 @@ $as_echo "yes: $jd_pcre_library_dir and
     jd_with_pcre_library="yes"
             PCRE_LIB=-L$jd_pcre_library_dir
     PCRE_LIB_DIR=$jd_pcre_library_dir
-    if test "X$jd_pcre_library_dir" = "X/usr/lib"
+    if test "X$jd_pcre_library_dir" = "X/usr/lib" -o "X$jd_pcre_library_dir" = "X/usr/lib64"
     then
       PCRE_LIB=""
     else
@@ -7302,7 +7302,7 @@ $as_echo "yes: $jd_onig_library_dir and
     jd_with_onig_library="yes"
             ONIG_LIB=-L$jd_onig_library_dir
     ONIG_LIB_DIR=$jd_onig_library_dir
-    if test "X$jd_onig_library_dir" = "X/usr/lib"
+    if test "X$jd_onig_library_dir" = "X/usr/lib" -o "X$jd_onig_library_dir" = "X/usr/lib64"
     then
       ONIG_LIB=""
     else
@@ -7554,7 +7554,7 @@ $as_echo "yes: $jd_png_library_dir and $
     jd_with_png_library="yes"
             PNG_LIB=-L$jd_png_library_dir
     PNG_LIB_DIR=$jd_png_library_dir
-    if test "X$jd_png_library_dir" = "X/usr/lib"
+    if test "X$jd_png_library_dir" = "X/usr/lib" -o "X$jd_png_library_dir" = "X/usr/lib64"
     then
       PNG_LIB=""
     else
@@ -7806,7 +7806,7 @@ $as_echo "yes: $jd_z_library_dir and $jd
     jd_with_z_library="yes"
             Z_LIB=-L$jd_z_library_dir
     Z_LIB_DIR=$jd_z_library_dir
-    if test "X$jd_z_library_dir" = "X/usr/lib"
+    if test "X$jd_z_library_dir" = "X/usr/lib" -o "X$jd_z_library_dir" = "X/usr/lib64"
     then
       Z_LIB=""
     else
@@ -8065,7 +8065,7 @@ $as_echo "yes: $jd_iconv_library_dir and
     jd_with_iconv_library="yes"
             ICONV_LIB=-L$jd_iconv_library_dir
     ICONV_LIB_DIR=$jd_iconv_library_dir
-    if test "X$jd_iconv_library_dir" = "X/usr/lib"
+    if test "X$jd_iconv_library_dir" = "X/usr/lib" -o "X$jd_iconv_library_dir" = "X/usr/lib64"
     then
       ICONV_LIB=""
     else
